util/tracing/detect/jaeger/jaeger.go

```diff
@@ -22,7 +22,7 @@ func jaegerExporter() (sdktrace.SpanExporter, error) {
 
 	endpoint := envOr("OTEL_EXPORTER_JAEGER_ENDPOINT", "http://localhost:14250")
 	host := envOr("OTEL_EXPORTER_JAEGER_HOST", "localhost")
-	port := envOr("OTEL_EXPORTER_JAEGER_PORT", "6832")
+	port := envOr("OTEL_EXPORTER_JAEGER_PORT", "6831")
```

Comment:
Is this change intentional?

Reply:
Yes. iirc one is for compress and the other uncompress. I've tested that 6831 works by default.